core/templates/dev/head/pages/exploration-editor-page/services/router.service.ts

```diff
@@ -46,6 +46,16 @@ angular.module('oppia').factory('RouterService', [
 
     var SLUG_GUI = 'gui';
     var SLUG_PREVIEW = 'preview';
+    // PREVIEW_TAB_WAIT_TIME_MSEC is the minimum duration to wait
+    // before calling _actuallyNavigate. This is done in order to
+    // ensure all pending changes are saved before navigating to
+    // the preview tab.
+    // _savePendingChanges triggers 'externalSave' event which
+    // will be caught by appropriate editors that will
+    // save pending changes. However, the autosave / saving of
+    // changelist is async. To allow autosave / saving of change
+    // list to complete. PREVIEW_TAB_WAIT_TIME_MSEC is provided.
```

Comment:
Change the period after "complete" into a comma.

Explain here why this can't be done using a promise ("it's async" isn't actually a sufficient reason).

Reply:
Done. Please let me know if the explanation is sufficient and clear.